core/src/kvs/ds.rs

```diff
@@ -1021,9 +1021,10 @@ impl Datastore {
 				Some(doc)
 			}
 			TableMutation::Def(_) => None,
-			TableMutation::SetPrevious(id, _old, new) => {
+			TableMutation::SetWithDiff(id, new, _operations) => {
 				let doc = Document::new(None, Some(id), None, new, Workable::Normal);
-				// TODO set previous value
+				// TODO(SUR-328): set previous value
+				warn!("We don't support applying patches as that was always on the user; This is behind a fflag");
```

Comment:
who is meant to read this log? There seems to be a lot of context implicit in here and users or even I may not understand what it means.